test/unit/app/lib/date-utils.test.js

```diff
@@ -1,11 +1,15 @@
 const { startandEndDate } = require('../../../../app/lib/date-utils.js')
 
-test('startandEndDate should return an object with start and end date', () => {
-  const date = '01/01/2022'
-  const result = startandEndDate(date)
-  const expectedStartDate = new Date(2022, 0, 1)
-  const expectedEndDate = new Date(2022, 0, 2)
+describe('date utils', () => {
+  describe('startandEndDate', () => {
+    it('should return an object with start and end date', () => {
```

Comment:
Prefer the flatter syntax of `test` personally.

Reply:
do you mean describe > describe > test? Or simply describe > test?

For the latter, you lose some grouping with tests against their parent function, is all.

Reply:
In my experience the use of `describe` leads to harder to read code and encourages sharing of scope. This is maybe worth a read https://kentcdodds.com/blog/avoid-nesting-when-youre-testing

so just `test` for me :)